manifest.json

```diff
@@ -50,7 +50,7 @@
     ],
     "content_scripts": [
         {
-            "js": [ "js/libs/jquery-3.2.1.js", "js/libs/contentScriptsAPIBridge.js", "js/tools.js", "js/plugins.js", "plugins/default.js", "js/hoverzoom.js"],
+            "js": [ "js/hls.js", "js/libs/jquery-3.2.1.js", "js/libs/contentScriptsAPIBridge.js", "js/tools.js", "js/plugins.js", "plugins/default.js", "js/hoverzoom.js"],
```

Comment:
This library is kinda large (~1MB) to include it unconditionally on all sites. If only Twitch uses it, why not include it on line 1692 with the twitch plugin?